python/pyspark/mllib/classification.py

```diff
@@ -594,19 +594,27 @@ def train(cls, data, lambda_=1.0):
 @inherit_doc
 class StreamingLogisticRegressionWithSGD(StreamingLinearAlgorithm):
     """
-    Run LogisticRegression with SGD on a batch of data.
-
-    The weights obtained at the end of training a stream are used as initial
-    weights for the next batch.
-
-    :param stepSize: Step size for each iteration of gradient descent.
-    :param numIterations: Number of iterations run for each batch of data.
-    :param miniBatchFraction: Fraction of data on which SGD is run for each
-                              iteration.
-    :param regParam: L2 Regularization parameter.
-    :param convergenceTol: A condition which decides iteration termination.
+    Train or predict a logistic regression model on streaming data. Training uses
+    Stochastic Gradient Descent to update the model based on each new batch of
+    incoming data from a DStream.
+
+    Each batch of data is assumed to be an RDD of LabeledPoints.
+    The number of data points per batch can vary, but the number
+    of features must be constant. An initial weight
+    vector must be provided.
+
+    :param stepSize:          Step size for each iteration of gradient descent.
```

Comment:
We shouldn't do vertical alignment. If in the future we add a new parameter with a long name, we have to change all lines. There are two options:

~~~
:param stepSize: Step size for each iteration of gradient descent.
~~~

or 

~~~
:param stepSize:
  Step size for each iteration of gradient descent.
~~~

I think the latter one is better because it doesn't affected by the length of the parameter name.